src/plugins/intel_cpu/tests/functional/single_layer_tests/augru_sequence.cpp

```diff
@@ -178,14 +194,13 @@ TEST_P(AUGRUSequenceCPUTest, CompareWithRefs) {
 
 namespace {
 /* CPU PARAMS */
-std::vector<std::map<std::string, std::string>> additionalConfig
-    = {{{InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16, InferenceEngine::PluginConfigParams::NO}},
-       {{InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16, InferenceEngine::PluginConfigParams::YES}}};
+std::vector<ov::AnyMap> additionalConfig = {{ov::hint::inference_precision()},
```

Comment:
Can it be empty?

What if use empty `{}` as `{InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16, InferenceEngine::PluginConfigParams::NO}` instead of `ov::hint::inference_precision(ov::element::f32)`?

Reply:
Good finding!
I think @zhaixuejun1993 should made a mistake here.